### ISSUE TITLE
Multi-label classification

### ISSUE THREAD

**Issue by Hosein47** [lucidrains/vit-pytorch#10]
How can one use this for multi-label classification?

**Comment by lucidrains** [comment]
@Hosein47 yea of course! it's just a separate loss function, and independent of the framework

for multi-label, you'll go with a binary cross entropy loss instead

**Comment by Hosein47** [comment]
Hmm, tnx. Which transformer method do you deem the best for multi-label classification? I mean Sinkhorn or routing or any other?

**Comment by lucidrains** [comment]
@Hosein47 They will all work for multi-label, since the difference is just in the loss function, not the architecture
